llvm-obs/obs-scripts/grant_permissions.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+mysql -u root -pTOPSECRET <<EOF
```

Comment:
maybe the TOPSECRET should be moved into a different place
https://docs.saltstack.com/en/latest/topics/best_practices.html#storing-secure-data ?

Reply:
For other I'll saved password in pillar but I think same would not be possible for a sh file, what do you think ? Also I've used sh file for now because it was not working with salt.states.mysql_grants or salts.states.mysql_query but I will try to make it work with salt itself.

Reply:
Could you create a fake pillar? thanks :)